step-templates/mysql-backup-database.json

```diff
@@ -0,0 +1,105 @@
+      "Octopus.Action.EnabledFeatures": ""
+    },
+    "Parameters": [
+      {
+        "Id": "7030ad54-4e10-4854-998a-ca91e3b490a3",
+        "Name": "createMySQLServerName",
```

Comment:
Can you namespace this parameter, e.g. `BackupMySQL.ServerName` or something similar?